aws_lambda_builders/workflows/provided_make/utils.py

```diff
@@ -0,0 +1,46 @@
+"""
+Commonly used utilities
+"""
+
+import os
+import platform
+import subprocess
+
+from aws_lambda_builders.utils import which
+
+
+class OSUtils(object):
```

Comment:
The famous Copy-paste OSUtils. We should make a generic one eventually

Reply:
yeah. we should.